XCUITests/TopTabsTest.swift

```diff
@@ -351,3 +351,21 @@ class TopTabsTestIphone: IphoneOnlyTestCase {
         XCTAssertTrue(app.buttons["TabTrayController.maskButton"].isEnabled)
     }
 }
+
+class TopTabsTestIpad: IpadOnlyTestCase {
+    func testUpdateTabCounter(){
+        
+        app/*@START_MENU_TOKEN@*/.buttons["New Tab"]/*[[".buttons[\"New Tab\"]",".buttons[\"TopTabsViewController.newTabButton\"]"],[[[-1,1],[-1,0]]],[1]]@END_MENU_TOKEN@*/.tap()
+        app/*@START_MENU_TOKEN@*/.buttons["New Tab"]/*[[".buttons[\"New Tab\"]",".buttons[\"TopTabsViewController.newTabButton\"]"],[[[-1,1],[-1,0]]],[1]]@END_MENU_TOKEN@*/.tap()
+        let numTab = app.buttons["Show Tabs"].value as? String
+        XCTAssertEqual("3", numTab)
+        app.collectionViews["Top Tabs View"].children(matching: .cell).matching(identifier: "home").element(boundBy: 1).buttons["Remove page — Open New Tab"].tap()
+        waitforExistence(app.buttons["Show Tabs"])
```

Comment:
Not sure if we need this line...that buttons is always there, can you check if removing it the test passes? Thanks!

Reply:
The test failed when I run  them for the first time without this line. Adding that line prevented them from failing. I think the tests were failing because of the animation that occurs after closing a tab.

Reply:
ah okay, then leave it there :)